app/views/qsets/show.html.erb

```diff
@@ -25,8 +25,13 @@
           <h4 class="panel-title">
             <div class="row valign">
               <div class="col-sm-11">
+                <div class="question-voting">
+                  <a class="upvote-button" data-vote="upvote" data-vote-qid="<%= question.id %>">Up</a><br>
+                  <div class="net-votes"><p id="<%= question.id %>"><%= question.votes_for - question.votes_against %></p><br></div>
```

Comment:
The <p> tag ID isn't specific enough -- HTML element ID should be unique on a page, and these paragraphs would have ID=4, 6, 7, etc... Maybe `id="question-<%= question.id %>-net-votes`?